index.html

```diff
@@ -8,231 +8,237 @@
-          </div>
-        </section>
-      </li>
-    </ul>
-  </section>
-  <section class="card-list-5">
-    <h3 class="about-me">
-      About Myself
-    </h3>
-    <p class="prg">
-      Hello I’m a software developer! I can help you build a product ,
-       feature or website Look through some of my work and experience!
-        If you like what you see and have a project you need coded,
-         don’t hestiate to contact me.
-    </p>
-    <h4 class="connect"><a href="#">LET'S CONNECT</a></h4>
-    <ul class="ul-list-icons-2">
-      <li><a href="https://twitter.com/KayongaChiteta3?t=gfILCjmltzGRZOx6FZ8-nQ&s=08)"><img src="./photos/icons/twitter-icon.png" alt="twitter-icom"></a></li>
-      <li><a href="https://www.linkedin.com/in/kayonga-chiteta-776949227"><img src="./photos/icons/Linkedin-icon.png" alt="linkedin-icon"></a></li>
-      <li><a href="#"><img src="./photos/icons/meadia.png" alt="m-icon"></a></li>
-      <li><a href="https://github.com/Kayonga99"><img src="./photos/icons/github.png" alt="github-icon"></a></li>
-      <li><a href="#"></a><img src="./photos/icons/icon-2.png" alt="icon"></li>
-    </ul>
-    <button class="btn-2" type="button">Get my resume</button>
-    <div class="lang-1">
-      <h6 class="lang">Languages</h6>  <a href="#"><img src="./photos/icons/d-arrow.png" alt="arrow icon"></a>
-    </div>
-    <div>
-      <ul>
-        <li class="tech">
-          <img class="tech-lang-2" src="./photos/Ellipsejs.png" alt="js photo">
-          <p class="tech-lang">JavaScript</p>
+          </section>
         </li>
-        <li class="tech">
-          <img class="tech-lang-2" src="./photos/Ellipse.png" alt="html photo">
-           <p class="tech-lang">HTML</p>
+        <li>
+          <section class="card-list">
+            <div class="desk-top">
+              <img class="animated" src="./photos/Snapshoot-Portfolio.png" alt="work1Snapshot" />
+              <img class="card" src="./photos/Snapshoot-yonga.png" alt="work1Snapshot" />
+              <div class="">
+                <h3 class="tonic-class">Tonic</h3>
+                <div class="canopy">
+                  <ul class="cards-ul">
+                    <li class="cano">CANOPY</li>
+                    <li class="info">Back End Dev</li>
+                    <li class="info">2015</li>
+                  </ul>
+                </div>
+                <p class="description">
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required.
+                </p>
+                <ul class="tools">
+                  <li class="tools-li">html</li>
+                  <li class="tools-li">css</li>
+                  <li class="tools-li">javaScript</li>
+                </ul>
+                <div class="view-project">
+                  <button class="btn" type="button">See Project</button>
+                </div>
+              </div>
+            </div>
+          </section>
         </li>
```

Comment:
- [ ] [OPTIONAL] Kindly increase the size of these cards to meet the Figma design
- Figma design
<img width="538" alt="Screen Shot 2022-03-25 at 14 02 26" src="https://user-images.githubusercontent.com/28307960/160117302-a10a6537-5a5f-4ae2-837a-bacd6ff534e5.png">
- Your design
<img width="1440" alt="Screen Shot 2022-03-25 at 13 53 28" src="https://user-images.githubusercontent.com/28307960/160117141-eea76c6d-c164-48fa-99fd-32a242953da9.png">